release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -264,14 +270,72 @@ submodule openconfig-pf-forwarding-policies {
         according to the IP destination).";
     }
 
+    leaf decap-network-instance {
+      type leafref {
+        // this must be an absolute reference to allow another NI
+        // to be referenced.
+        path "/network-instances/network-instance/config/name";
+      }
+      description
+        "This leaf is mutually exclusive with ../network-instance.
+        This leaf is normally used with NIs (network instances) that are
+        populated by SDN controllers (e.g. via gRIBI), where we want:
+          - The decapsulation decision to be controlled by the
+            controllers.
+          - The new header after decapsulation is to be looked up in
+            another NI, and the NI is chosen by fields (e.g. source IP,
+            DSCP, etc) that are not used for route resolution.
```

Comment:
Does this help further clarify the use case?

```suggestion
          - The new header after decapsulation is to be looked up in
            another NI.  The target NI is set by the SDN controller
            possibly using fields that are not use for route resolution
            (e.g. source IP, DSCP, etc).
```

Reply:
It might not be SDN controllers/pipeline that populate the fields. It might be config generation pipelines. Therefore, I didn't plan to call it out specifically.

Reply:
SGTM